src/rod/sdf/element.py

```diff
@@ -47,7 +47,7 @@ def serialize_float(data: float) -> str:
     @staticmethod
     def serialize_list(data: list[float]) -> str:
         assert isinstance(data, list)
-        return " ".join(np.array(data, dtype=str))
+        return " ".join(np.array(data, dtype=float).astype(str))
```

Comment:
This is unfortunately not equivalent, we need to cast elements of `data` to `float`. This can be regardless be done with a custom lambda. Done in https://github.com/ami-iit/rod/pull/40/commits/cc68de03c831ab22492154063c2baf81a0fa5251.